documentation/sphinx/source/release-notes/release-notes-700.rst

```diff
@@ -31,7 +31,9 @@ Fixes
 Status
 ------
 * Added ``commit_batching_window_size`` to the proxy roles section of status to record statistics about commit batching window size on each proxy. `(PR #4735) <https://github.com/apple/foundationdb/pull/4735>`_
-
+* Added ``cluster.bounce_impact`` section to status to report if there will be any extra effects when bouncing the cluster, and if so, the reason for those effects. `(PR #4770) <https://github.com/apple/foundationdb/pull/4770>`_
+* Added ``fetched_versions`` to the storage metrics section of status to report how fast a storage server is catching up in versions. `(PR #4770) <https://github.com/apple/foundationdb/pull/4770>`_
+* Added ``fetches_from_logs`` to the storage metrics section of status to report how frequently a storage server fetches updates from transaction logs. `(PR #4770) <https://github.com/apple/foundationdb/pull/4770>`_
```

Comment:
This is not exactly a problem with this PR, but I'm just noticing that here that you've updated release-notes-700.rst, which is what was updated on master. This was also backported to release-6.3, and that change includes updates to release-notes-630.rst.

We need to update release-notes-630.rst on master and backport them here as well, and I think the easiest thing to do is to open a new PR on master, get it merged, and then cherry pick it into this PR alongside the other changes.

Reply:
Oh, sorry, I wasn't aware of the policy about updating the release notes. Let me update release-notes-630.rst  and include that commit here. I will create a new PR for making the corresponding change in master (I don't want to create yet another PR to cherry-pick that to 7.0.).

Reply:
Done.